Lib/idlelib/autocomplete_w.py

```diff
@@ -246,7 +246,7 @@ def winconfig_event(self, event):
         acw.wm_geometry("+%d+%d" % (new_x, new_y))
 
         if platform.system().startswith('Windows'):
-            # See issue 15786. When on windows platform, Tk will misbehave
+            # See issue 15786. When on windows platform, Tk will misbehaive
```

Comment:
Here it looks like the PR is introducing a typo. Did something go wrong with the separate idlelib change?

Reply:
Thank you, my bad, incorrect rebase. Since current master already merged that changes separately, I now excluded idlelib revert-patches from this request.